meteor/client/ui/Shelf/DashboardPanel.tsx

```diff
@@ -719,6 +697,7 @@ export function isAdLibOnAir(
 	unfinishedTags: IDashboardPanelTrackedProps['unfinishedTags'],
 	adLib: AdLibPieceUi
 ) {
+	console.log('isAdLibOnAir', adLib.name, adLib._id, adLib.currentPieceTags, unfinishedTags)
```

Comment:
This should be removed, otherwise, unless the performance is hurt, it looks good.